### DISCUSSION TITLE
how to change the locale?

### DISCUSSION THREAD

**Discussion by JSchmidt63** [energye/energy#9] [Q&A]
I want to change the locale ant the cef?

Maybe at english or german?

**Comment by JSchmidt63** [comment]
i use that:
applicationConfig.SetLanguage(LANGUAGE_en_US)
but i can't see the right-click menu in en_US, but in zn_CN

Cheers :-)

`package main

import (
	"github.com/energye/energy/cef"
	. "github.com/energye/energy/consts"
)

func main() {
	//全局初始化 每个应用都必须调用的
	cef.GlobalInit(nil, nil)
	applicationConfig := cef.NewApplicationConfig()
	//可选配置: VF组件窗口需要指定该配置
	applicationConfig.SetLanguage(LANGUAGE_en_US)  // i use that
	applicationConfig.SetRemoteDebuggingPort(8888) //远程端口方式, 需自定义端口号
	//创建应用
	cefApp := cef.NewApplication(applicationConfig)
	//主窗口的配置
	//指定一个URL地址，或本地html文件目录
	cef.BrowserWindow.Config.Url = "https://mozilla.org"
	//chromium 配置
	config := cef.BrowserWindow.Config.ChromiumConfig()
	config.SetEnableMenu(true)     //启用右键菜单
	config.SetEnableDevTools(true) //启用开发者工具
	//运行应用
	cef.Run(cefApp)
}`

![image](https://github.com/energye/energy/assets/25293774/8903a120-8451-444f-ac1d-c021a5d92dda)

**Comment by JSchmidt63** [comment]
well, i see the hard coded in "zn_CN" in the "cef-events-callback.go", see in the line 233 and so

**Reply by sxmxta** [reply]
Indeed, the Chinese hard coding used in the menu here is not internationalized

It will gradually improve in the future

**Comment by sxmxta** [comment]
I18n support added
